package.json

```diff
@@ -9,38 +9,43 @@
   "repository": "gulpjs/remove-bom-stream",
   "license": "MIT",
   "engines": {
-    "node": ">= 0.10"
+    "node": ">= 10.13.0"
   },
   "main": "index.js",
   "files": [
     "LICENSE",
     "index.js"
   ],
   "scripts": {
-    "lint": "eslint index.js test/ && jscs index.js test/",
+    "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "mocha --async-only",
-    "cover": "istanbul cover _mocha",
-    "coveralls": "npm run cover && istanbul-coveralls"
+    "test": "nyc mocha --async-only"
   },
   "dependencies": {
-    "safe-buffer": "^5.1.0",
     "remove-bom-buffer": "^3.0.0",
-    "through2": "^2.0.3"
+    "safe-buffer": "^5.2.1",
```

Comment:
We'll want to drop safe-buffer completely, it was a holdover for the old buffer support